gdcdictionary/schemas/follow_up.yaml

```diff
@@ -956,9 +958,196 @@ properties:
+        - Hepatitis C Infection
+        - Hepatitis, Chronic
+        - Hereditary Non-polyposis Colon Cancer
+        - Herpes
+        - Herpes Zoster
+        - High Grade Liver Dysplastic Nodule
+        - HIV / AIDS
+        - Hodgkin Lymphoma
+        - Human Papillomavirus Infection
+        - HUS/TTP
+        - Hypercalcemia
+        - Hypercholesterolemia
+        - Hyperglycemia
+        - Hyperlipidemia
+        - Hypertension
+        - Hypospadias
+        - Hypothyroidism
+        - Inflammatory Bowel Disease
+        - Insulin Controlled Diabetes
+        - Interstitial Pneumontis or ARDS
+        - Intraductal Papillary Mucinous Neoplasm
+        - Iron Overload
+        - Ischemic Heart Disease
+        - ITP
+        - Joint Replacement
+        - Kidney Disease
+        - Li-Fraumeni Syndrome
+        - Liver Cirrhosis (Liver Disease)
+        - Liver Toxicity (Non-Infectious)
+        - Low Grade Liver Dysplastic Nodule
+        - Lupus
+        - Lymphamatoid Papulosis
+        - Lymphocytic Meningitis
+        - Lynch Syndrome
+        - MAI
+        - Malaria
+        - Metabolic Syndrome
+        - Methicillin-Resistant Staphylococcus aureus (MRSA)
+        - Myasthenia Gravis
+        - Mycobacterium avium Complex
+        - Myocardial Infarction
+        - Neuroendocrine Tumor
+        - Nonalcoholic Steatohepatitis
+        - Obesity
+        - Organ transplant (site)
+        - Osteoarthritis
+        - Osteoporosis or Osteopenia
+        - Other
+        - Other Cancer Within 5 Years
+        - Other Nonmalignant Systemic Disease
+        - Other Pulmonary Complications
+        - Pain (Various)
+        - Pancreatitis
+        - Peptic Ulcer (Ulcer)
+        - Peripheral Neuropathy
+        - Peripheral Vascular Disease
+        - Peutz-Jeghers Disease
+        - Pneumocystis Pneumonia
+        - Polycystic Ovarian Syndrome (PCOS)
+        - Pregnancy in Patient or Partner
+        - Primary Sclerosing Cholangitis
+        - Psoriasis
+        - Pulmonary Fibrosis
+        - Pulmonary Hemorrhage
+        - Renal Dialysis
+        - Renal Failure (Requiring Dialysis)
+        - Renal Insufficiency
+        - Rheumatoid Arthritis
+        - Rheumatologic Disease
+        - Rubinstein-Taybi Syndrome
+        - Sarcoidosis
+        - Seizure
+        - Shingles
+        - Sjogren's Syndrome
+        - Sleep apnea
+        - Smoking
+        - Staph Osteomyelitis
+        - Staphylococcus aureus
+        - Steatosis
+        - Stroke
+        - Syphilis
+        - Thyroid Disease, Non-Cancer
+        - Transient Ischemic Attack
+        - Treponema pallidum
+        - Tuberculosis
+        - Turcot Syndrome
+        - Tyrosinemia
+        - Ulcerative Colitis
+        - Urinary Tract Infection
+        - Varicella Zoster Virus
+        - Wagr Syndrome
+        - Unknown
+        - Not Reported
+
   comorbidity:
     $ref:
-      - "_terms.yaml#/comorbidity/common"
+      - "_terms.yaml#/comorbidities/common"
```

Comment:
qn. if we are deprecated comorbidity, is it better to remove the ref to terms or update the ref (which you do here)?

Reply:
I updated the ref to get the travis checks to pass since I think comorbidity was removed from terms file. Not sure what the precedent is for this, I can try removing the ref and see if it makes a difference at all.

Reply:
@bullenc and @wteouchicago I'm a little confused... we're looking at the old property here, right - the deprecated property? I think in the past we have left that as is (outside of adding it to the deprecated list) until it's ready to be removed (i.e. after the data migration to the new property). Not sure if that's helpful, but I do think we should create a new entry in the terms file for the plural version so that it matches the follow-up schema.

Reply:
@tlicht3 I'm not 100% familiar with what we did the past, but yeah, I had expected something like:
- just add the old singular comorbidity to deprecated list
- add the new plural comorbidities with all its enums and also to terms file

But I defer to everyone else on this. :)

Reply:
@tlicht3 and @wteouchicago so comorbidity is in the deprecated list, context for the above changes: I just changed (then deleted) the reference for the deprecated comorbidity property since the _comorbidity_ entry in the _terms.yaml file was removed- I assumed this was intentional; if so, checks would break because the property in the follow_up.yaml file pointed to a non-existent term in the _terms.yaml file- so thus I changed to an existing one ("_terms.yaml#/comorbidities/common") for checks to go thru, and then I just removed it altogether after Wendy's comment. 

It sounds like the removal of the comorbidity entry in the _terms.yaml file was erroneous and precedent was to leave everything alone for deprecated properties.

Perhaps then we should:

> add back in entry for  comorbidity in _terms.yaml file
> add back in "_terms.yaml#/comorbidity/common" reference for comorbidity

Reply:
that sounds right to me, though @wwysoc2 can probably confirm as well.

Reply:
Do any of these options that you would like (or leave it as is).  As long as "comorbidities" points to "comorbidities", that's okay with me.  "comorbidity" is deprecated, and therefore won't show up in the viewer, so it shouldn't matter where the ref file points.

Reply:
this all makes sense to me - i thought we needed the old term for the test to pass... i think i now understand that if we have the ref and there's nothing to reference in terms file it will error out, but there's nothing wrong with removing BOTH the ref and the term in the terms file.

Reply:
Ok I will leave it as is. Thanks for the input all!